io.freetubeapp.FreeTube.yml

```diff
@@ -7,6 +7,7 @@ base: org.electronjs.Electron2.BaseApp
 base-version: '21.08'
 command: run.sh
 separate-locales: false
+skip-arches: arm
```

Comment:
This line can be removed after https://github.com/FreeTubeApp/FreeTube/pull/2114 is merged